src/rabbit_auth_backend_ip_range.erl

```diff
@@ -27,6 +28,9 @@ description() ->
 user_login_authorization(_Username) ->
     {ok, none}.
 
+user_login_authorization(_Username, _AuthProps) ->
```

Comment:
This function only exists in `master`. We'll need to omit it when backport for `v3.7.x`.

Reply:
Having the function there won't matter since it'll be unused, correct? Also, should we create a `3.7.x` branch in this repo? I figured that after I merge this that point would become `3.7.x`

Reply:
Given it's a tier-2 plugin (i.e. not distributed with the broker), it would be awesome if there was a single codebase for 3.7.x and 3.8.x/`master` if possible. And if there is something we can do in RabbitMQ to allow that, then we should try to do it.

Reply:
OK, sounds good. I will leave this repo as is since having an extra function exported won't (shouldn't) matter.